backend/src/models/base/base.py

```diff
@@ -9,6 +10,14 @@
 class BaseModel:
     __abstract__ = True
 
+    metadata = MetaData(naming_convention={
+        "ix": "ix_%(column_0_label)s",
+        "uq": "uq_%(table_name)s_%(column_0_name)s",
+        "ck": "ck_%(table_name)s_`%(constraint_name)s`",
+        "fk": "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
+        "pk": "pk_%(table_name)s"
+    })
+
     @classmethod
     @declared_attr
     def __tablename__(cls):
```

Comment:
you can remove this stuff and just type `__tablename__`  class attribute in every other tables. I think it would be better